tools/windows/install-help/cal/PropertyView.cpp

```diff
@@ -0,0 +1,85 @@
+#include "stdafx.h"
+
+void parseKeyValueString(const std::wstring kvstring, std::map<std::wstring, std::wstring> &values)
+{
+    // Parse values out of property
+    std::wstringstream ss(kvstring);
+    std::wstring token;
+
+    while (std::getline(ss, token))
+    {
+        // 'token' contains "<key>=<value>"
+        std::wstringstream instream(token);
+        std::wstring key, val;
+        if (std::getline(instream, key, L'='))
+        {
+            trim_string(key);
+            std::getline(instream, val);
+            trim_string(val);
+            if (!key.empty() && !val.empty())
+            {
+                values[key] = val;
+            }
+        }
+    }
+}
+
+bool PropertyView::present(const std::wstring &key) const
+{
+    return this->values.count(key) != 0 ? true : false;
```

Comment:
Would not this work properly
```suggestion
    return this->values.count(key) != 0;
```